merged-packages/design-tokens/package.json

```diff
@@ -17,9 +17,14 @@
   "license": "(MIT OR Apache-2.0)",
   "exports": {
     ".": {
-      "types": "./dist/types/index.d.ts",
-      "import": "./dist/index.mjs",
-      "require": "./dist/index.js"
+      "import": {
+        "types": "./dist/index.d.mts",
+        "default": "./dist/index.mjs"
+      },
+      "require": {
+        "types": "./dist/index.d.cts",
+        "default": "./dist/index.cjs"
+      }
```

Comment:
Updating to align with packages we are not changing the exports because we are still exporting the types folder on line `34`